lib/floe/workflow/runner/docker.rb

```diff
@@ -74,7 +74,9 @@ def success?(runner_context)
 
         def output(runner_context)
           output = docker!("logs", runner_context["container_ref"], :combined_output => true).output
-          runner_context["output"] = output
+          return unless output.index(OUTPUT_MARKER)
```

Comment:
> Do we want to raise an error here so the user knows something is wrong?

No because I don't want to assume that the container has finished at this point, it could be running and simply not printed all output yet.

Reply:
I didn't think we called `output` until after the container has been marked as done
and `output` was only called once. never an intermediate value.

Hmm... that doesn't make debugging that useful